envi/config.py

```diff
@@ -180,10 +180,16 @@ def parseConfigOption(self, optstr):
             valstr = str(int(valstr, 16))
 
         if not (valstr.startswith('"') and valstr.endswith('"')):
-            try:
-                int(valstr)
-            except:
-                valstr = '"' + valstr + '"'
+            if valstr.lower() == 'true':
```

Comment:
does this make sense?  if you want something different, i'm not clear on what it is.  i looked up ast.literal_eval and i'm not sure what we'd gain, so please explain what success looks like.